flintrock/flintrock.py

```diff
@@ -226,6 +226,8 @@ def cli(cli_context, config, provider):
 @click.option('--ec2-ebs-optimized/--no-ec2-ebs-optimized', default=False)
 @click.option('--ec2-instance-initiated-shutdown-behavior', default='stop',
               type=click.Choice(['stop', 'terminate']))
+@click.option('--ec2-user-data', default='',
+              help="Path to userdata script file for accessing nodes.")
```

Comment:
I don't understand what you mean here by "for accessing nodes". Perhaps we could just phrase this as follows?

> Path to EC2 user data script that will run on instance launch.